diagnostic.md

```diff
@@ -45,7 +45,8 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+it runes 4 times.
+value  = 5
```

Comment:
The loop runs 3 times and x is equal to 5 once it's finished.  On the first iteration, x = 1 + 1 = 2. On the second iteration, x = 2 + 2 = 4. On the third iteration, x = 4 + 4 = 8. The loop no longer runs since x < 5 is no longer satisfied.